Add a nix flake for snazy (#4)
chmouel/snazy: README.md

yay -S snazy-bin
```

### [Nix/NixOS](https://nixos.org/)

This repository comes with a `flake` (see [NixOS Wiki on
Flakes](https://nixos.wiki/wiki/Flakes)).

> If you have the `nix flake` command enable (currenty on

Reviewer: ```suggestion
If you have the `nix flake` command enabled (currenty on
```

nixos-unstable, `nixos-version` >= 22.05)

```shell
nix run github:chmouel/snazy -- --help # your args are here
```

> You can also use to test and develop on the repository.

Reviewer: ```suggestion
You can also use it to test and develop the source code: 
```
